UPGRADING.rst

```diff
@@ -11,6 +11,23 @@ reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
 
+#2950 Move auth and cart service to attic
+=========================================
+
+1. Before upgrading to this commit, run ::
```

Comment:
Why is this necessary?

Reply:
Terraform cant figure out how to properly teardown the infra due to a `Cycle Error`:

<img width="1920" alt="Screen Shot 2021-04-15 at 11 17 14 AM" src="https://user-images.githubusercontent.com/1947534/114918955-897d7a80-9ddc-11eb-96b1-0d1687944b71.png">
https://github.com/DataBiosphere/azul/pull/2969#issuecomment-819896848

Reply:
OK, I've tried my mightiest yesterday to avoid this workaround, to no avail.

Reply:
https://github.com/DataBiosphere/azul/issues/2992